new version & changelog (#1514)
ScilifelabDataCentre/dds_web: CHANGELOG.rst

Changelog
==========

.. _2.6.2:

2.6.2 - 2023-02-13
~~~~~~~~~~~~~~~~~~~~~~~

- New features:

>     - Documentation is now available at `/documentation/` Some endpoints are not properly documented due to openApi limitations.

Reviewer: ```suggestion
    - Documentation is now available at `https://delivery.scilifelab.se/documentation/`. Some endpoints do not comply with the openAPI standards and are not properly documented.
```
I also spotted a typo in the previous entry, so maybe take the opportunity to fix it
```
Listing users invites will now show if the invite is for Project Owner.
```
